src/tuplet.ts

```diff
@@ -98,24 +97,23 @@ export class Tuplet extends Element {
       throw new RuntimeError('BadArguments', 'No notes provided for tuplet.');
     }
 
-    this.options = Vex.Merge({}, options);
+    this.options = { ...options };
     this.notes = notes;
-    this.num_notes = 'num_notes' in this.options ? this.options.num_notes : notes.length;
+    this.num_notes = this.options.num_notes ? this.options.num_notes : notes.length;
```

Comment:
This will break for `num_notes == 0`, no?

Reply:
`undefined` used